driver/bpf/fillers.h

```diff
@@ -2083,6 +2083,39 @@ static __always_inline bool get_exe_writable(struct task_struct *task)
 	return false;
 }
 
+static __always_inline bool get_exe_upper_layer(struct task_struct *task)
+{
+	struct file *exe_file;
+	struct mm_struct *mm;
+
+	mm = _READ(task->mm);
+	exe_file = _READ(mm->exe_file);
+	if (!exe_file)
+	{
+		return false;
+	}
+
+	struct inode *exe_inode = _READ(exe_file->f_inode);
+	struct super_block *sb = _READ(exe_inode->i_sb);
```

Comment:
I see that basically until this loc, this functions shares 100% of the code with `get_exe_writable`.
I am not sure; is it costly to call all of these?
Can't we have a single function to extract both booleans?